eng/pipelines/runtime.yml

```diff
@@ -288,7 +288,7 @@ extends:
             testGroup: innerloop
             timeoutInMinutes: 120
             nameSuffix: NativeAOT
-            buildArgs: -s clr.aot+host.native+libs -rc $(_BuildConfig) -lc Release -hc Release
+            buildArgs: -s clr.aot+host.native+libs+tools.illink -c $(_BuildConfig) -rc $(_BuildConfig) -lc Release -hc Release
```

Comment:
What does -c do when we also specify -rc and -lc?

Reply:
The -c probably applies to the tools.illink subset.

Reply:
You could simplify this by removing the `-rc` switch.